### ISSUE TITLE
app_admin Role

### ISSUE THREAD

**Issue by nelsonic** [dwyl/auth#109]
If person authenticates on Auth without a Redirect URL Grant them "**app_admin**" role which allows them to administer their _own_ app(s) but is not "superadmin" or "admin" role. 

# Todo

+ [x] Add **`app_admin`** role to `priv/repo/default_roles.json`:
```json
  {
    "name": "app_admin", 
    "desc": "Can manage their own App(s).", 
    "person_id": "1",
    "id": "8",
    "permissions": "manage_apps, create_content, upload_images, edit_own_content, delete_own_content, invite_people"
  },
```
+ [ ] Assign **`app_admin`** role to people who authenticate _directly_ i.e. they don't have a redirect URL. Which means they are attempting to create/manage their own App(s).

I don't strictly know if this is necessary ...
Basically everyone who is _not_ a **`superadmin`** is an **`app_admin`** if they choose to be.
So we can just restrict the **`superadmin`** routes/content and everything else is accessible to anyone who should be _allowed_ to see it. It's just for the purposes of clarity I guess ... 
What do _you_ think? 💭

**Comment by nelsonic** [comment]
https://github.com/dwyl/auth/blob/e5d8c5e19306f7c07c0856b12e175df99056810f/priv/repo/default_roles.json#L51-L57

**Comment by nelsonic** [comment]
going to hold off on this for now as it doesn't add any value.
